packages/core/test/test-util/test-services/v2/test-service/BatchRequest.ts

```diff
@@ -4,7 +4,16 @@
  * This is a generated file powered by the SAP Cloud SDK for JavaScript.
  */
 import { CreateRequestBuilder, DeleteRequestBuilder, GetAllRequestBuilder, GetByKeyRequestBuilder, ODataBatchChangeSet, ODataBatchRequestBuilder, UpdateRequestBuilder } from '../../../../../src';
-import { TestEntity, TestEntityMultiLink, TestEntityOtherMultiLink, TestEntityLvl2MultiLink, TestEntitySingleLink, TestEntityLvl2SingleLink, TestEntityCircularLinkParent, TestEntityCircularLinkChild, TestEntityEndsWith, TestEntityEndsWithSomethingElse } from './index';
+import { TestEntity } from './TestEntity';
+import {  TestEntityMultiLink} from './TestEntityMultiLink'
```

Comment:
shouldn't be a lint issue?

Reply:
Actually I did a test:
1. open `packages/core/src/odata/common/request/odata-request.ts`
1. change `import { buildCsrfHeaders } from '../../../header-builder/csrf-token-header';` to `import {  buildCsrfHeaders} from '../../../header-builder/csrf-token-header';`
1. run lint and it will fix the issue for you.

Now the question is why this file or this case is ignored?

Reply:
ok, I see this file is in the scope of `packages/core/test/test-util/test-services` that is ignored.

Reply:
not sure the reason though, but please feel free to fix the imports in this file.

Reply:
yes, but the test-service is excluded by the linter. I just realized that this is auto generated code. So I added this to the follow up issue of adjust to the generator. It should import from the files and not from the index.